apps/klasses/templates/klasses/klass_header.html

```diff
@@ -1,5 +1,16 @@
 {% load static %}
 
+<style>
+    .condensed {
+            margin: 0.5vh 0 0 0 !important;
+            padding: 0.5vh 0 0 0 !important;
+        }
+
```

Comment:
Thanks for consolidating these styles into a single location! This should improve the maintainability of the project 😁

Reply:
Welcome!